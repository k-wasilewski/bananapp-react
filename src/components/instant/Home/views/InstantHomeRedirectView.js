import React, {Component} from "react";
import {Redirect} from 'react-router-dom';

class InstantHomeRedirectView extends Component {
    render() {
        return (
            <Redirect to={{
                pathname: '/results',
                state: { prediction: this.props.prediction,
                    img: this.props.img}
            }}/>
        )
    }
}
export default InstantHomeRedirectView;