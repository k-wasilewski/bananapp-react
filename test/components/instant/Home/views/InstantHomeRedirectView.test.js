import React from 'react';
import renderer from 'react-test-renderer';
import store from "../../../../../src/redux/store";
import {BrowserRouter} from "react-router-dom";
import InstantHomeRedirectView from "../../../../../src/components/instant/Home/views/InstantHomeRedirectView";
import {Provider} from "react-redux";

describe("InstantHomeRedirectView rendering specification", () => {
    it('InstantHomeRedirectView is rendered', () => {
        const component = renderer.create(
            <Provider store={store}>
                <BrowserRouter>
                    <InstantHomeRedirectView />
                </BrowserRouter>
            </Provider>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});