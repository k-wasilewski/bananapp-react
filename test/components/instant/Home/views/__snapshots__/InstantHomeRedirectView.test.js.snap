// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstantHomeRedirectView rendering specification InstantHomeRedirectView is rendered 1`] = `null`;
